great_expectations/rule_based_profiler/data_assistant_result/data_assistant_result.py

```diff
@@ -1970,53 +2014,205 @@ def _get_expect_domain_values_to_be_between_line_chart(
                 x=batch_plot_component.plot_on_axis(),
                 y=min_value_plot_component.plot_on_axis(),
                 y2=alt.Y2(
-                    max_value_plot_component.name, title=metric_plot_component.title
+                    max_value_plot_component.name, title=max_value_plot_component.title
                 ),
             )
             .properties(title=title)
         )
 
-        line: alt.Chart = DataAssistantResult._get_line_chart(
-            df=df,
-            metric_plot_component=metric_plot_component,
+        if "quantiles" in df.columns:
+            lower_limit = lower_limit.encode(detail="quantiles")
+            upper_limit = upper_limit.encode(detail="quantiles")
+            band = band.encode(detail="quantiles")
+
+        metric_name: str
+        predicate: Union[bool, int]
+        anomaly_coded_line: alt.Chart
+        anomaly_coded_lines: List[alt.Chart] = []
+        for metric_plot_component in metric_plot_components:
+            # encode point color based on anomalies
+            metric_name = metric_plot_component.name
+            predicate = (
+                (alt.datum.min_value > alt.datum[metric_name])
+                & (alt.datum.max_value > alt.datum[metric_name])
+            ) | (
+                (alt.datum.min_value < alt.datum[metric_name])
+                & (alt.datum.max_value < alt.datum[metric_name])
+            )
+            point_color_condition: alt.condition = alt.condition(
```

Comment:
You can get rid of the type annotation here since the construction is explicit here.

Reply:
I agree with this ✅